### DISCUSSION TITLE
Custom Web Service Listener - 404 Not Found: Invalid Request

### DISCUSSION THREAD

**Discussion by alexzeca** [nextgenhealthcare/connect#4928] [Q&A]
Hi, I'm trying to implement a Custom Web Service Listener based on the example described by narupley in the old forum in this topic (https://forums.mirthproject.io/forum/mirth-connect/support/9842-mirth-3-0-web-service-question) and having everything exactly the same code gives me the following error when accessing wsdl:

404 Not Found: Invalid Request

Mirth Connect 3.5.2 with Java 1.8.0_311

Any clues? What is intended is to receive an HL7v2 message in XML.

Thanks in advance.

**Comment by jonbartels** [comment]
Are there any messages in `mirth.log` related to the issue?

**Reply by alexzeca** [reply]
No, there aren't.

Web Service Listener config:
![image](https://user-images.githubusercontent.com/45019180/146961158-a4b7344a-0776-43ab-b457-5b007ef5e5ce.png)
And .jar file:
[AcceptXMLMessage.zip](https://github.com/nextgenhealthcare/connect/files/7756607/AcceptXMLMessage.zip)
